drivers/misc/ipts/companion.c

```diff
@@ -209,3 +209,22 @@ int ipts_request_firmware_config(struct ipts_info *ipts,
 	return ret;
 
 }
+
+bool ipts_needs_no_feedback(void)
+{
+	bool ret;
+
+	// Make sure that access to the companion is synchronized
+	mutex_lock(&ipts_companion_lock);
+
+	// If the companion is ignored, or doesn't exist, assume that
+	// the device doesn't need no_feedback enabled
+	if (ipts_modparams.ignore_companion || ipts_companion == NULL)
+		ret = false;
+	else
+		ret = ipts_companion->needs_no_feedback(ipts_companion);
```

Comment:
I like the idea of moving the device quirks to the companion driver, but can we do that a bit more generic? Currently adding a new quirk would require adding a function to the companion driver interface. I'd prefer having a generic quirk function that returns a `uint64_t` bitfield and something like `#define IPTS_QUIRK_NO_FEEDBACK BIT(0)`.